docs/reference/intro.asciidoc

```diff
@@ -1,68 +1,97 @@
 [[elasticsearch-intro]]
-== What is {es}?
+== {es} basics
+
+This guide covers the essential basics for getting started with {es}.
+If you'd prefer to get started with code examples, jump to our <<quickstart,quickstarts>>.
```

Comment:
FYI the new "quickstarts" section is all about running code, and not setting up local dev as they are now separate concerns

We chatted about how we want to get a better title for that "quickstarts" section too

modulo that, everything in this PR can be reworded more gooder

Reply:
I'll take suggestion 1 for the moment

Reply:
will go with "If you'd prefer to start working with {es} right away, then jump to our <<quickstart,quickstarts>> for hands-on code examples."

Reply:
where did the local dev go?

Reply:
right, there's a prerequisites section in quickstarts overview that links to local dev and deployment options, but could spell it out here too 👍

Reply:
`If you'd prefer to start working with {es} right away, set up a <<run-elasticsearch-locally,local dev environment>> and jump to our <<quickstart,hands-on code examples>> .`

Reply:
I find it a bit odd that throughout the docs, we say "our" examples, "our" tutorials... "we" shouldn't pull focus in things that aren't really about Elastic. 🙂  I'd just say "some" instead of "our"

Reply:
I'll do my part and remove "our" usages in this PR 🫡